src/main/java/com/cicdlectures/menuserver/controller/MenuController.java

```diff
@@ -37,4 +41,9 @@ public List<MenuDto> listMenus() {
   public MenuDto createMenu(@RequestBody MenuDto menu) {
     return createMenuService.createMenu(menu);
   }
+
```

Comment:
Vous n'avez pas donné de description à votre fonction, ce qui est, selon mes standards (qui sont plutôt élevés), inadmissible. Veillez donc à ne surtout pas l'admettre, et à rajouter (en toute discrétion) une description à cette fonction. Bien cordialement, Enzo le plus beau.
```suggestion
"""
Fonction complètement nulle de Hiba (aussi complètement nulle) qui permet de façon presque totalement non-fonctionnelle de supprimer des menus.
"""
```